.github/workflows/acceptance_test.yml

```diff
@@ -116,8 +116,8 @@ jobs:
       - name: Set URL matrix
         id: set-matrix
         run: |
-          python3 scripts/mobility-database-harvester/harvest_latest_versions.py -d scripts/mobility-database-harvester/datasets_metadata -l gtfs_latest_versions.json -o gtfs_older_versions.json
-          DATASETS=$(jq . scripts/mobility-database-harvester/datasets_metadata/latest_versions.json -c)
+          python3 scripts/mobility-database-harvester/harvest_latest_versions.py -d scripts/mobility-database-harvester
+          DATASETS=$(jq . scripts/mobility-database-harvester/latest_versions.json -c)
```

Comment:
I'm curious why the jq line is need here.  Isn't the data already compact json when output from the python script?

Reply:
I finally removed that line after reading your other comments. Now using stdout only.